### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by WojciechGdansk**
Updated link to download as previous doesnt work any more

### CODE REVIEW

File: pythonforandroid/recipes/freetype/__init__.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class FreetypeRecipe(Recipe):
     """
 
     version = '2.10.1'
-    url = 'https://download.savannah.gnu.org/releases/freetype/freetype-{version}.tar.gz'  # noqa
+    url = 'https://bigsearcher.com/mirrors/nongnu/freetype/freetype-2.10.1.tar.gz'  # WojciechGdansk
```
Comment (AndreMiras) [inline pythonforandroid/recipes/freetype/__init__.py:28]: It should be `{version}` also I'm not sure what's the `WojciechGdansk` comment about

**Comment by T-Dynamos** [comment]
Actually nothing seems wrong with current link.

**Comment by AndreMiras** [comment]
Yes indeed, I was able to wget https://download.savannah.gnu.org/releases/freetype/freetype-2.10.1.tar.gz

**Comment by WojciechGdansk** [comment]
I swear it didn't work today

**Pull request closed without merging**